### ISSUE TITLE
[Crash]: Ender Io and Ars additions crash

### ISSUE THREAD

**Issue by Cebio** [AllTheMods/ATM-10#746]
### Possible Fixes

Yes

### Modpack Version

0.66

### What happened?

[crash-2024-10-12_01.11.32-fml.txt](https://github.com/user-attachments/files/17347788/crash-2024-10-12_01.11.32-fml.txt)

The 2 mods i mentioned in title crashes while trying to load server. I just removed them from the mod file and the server started properly but i dont know if it gonna cause any problem in world's future. Is there anything i can do the make them work? My crashlog is in the mention.

### Crash Log

https://gist.github.com/Cebio/7c4c121ed9950a0d872e178559c928e7

**Comment by Rover656** [comment]
Ender IO and Ars Additions have problems with when the PC is set to Turkish. Ender IO update is not yet ready and unsure about the status of Ars. Apologies for the inconvenience.

**Comment by Cebio** [comment]
Yeah i posted also to spesifically mods github and they helped me with the problem. Thanks for the headsup anyway!
